src/styles/partials/components/_progresstabs.scss

```diff
@@ -0,0 +1,80 @@
+.dg_progress-tabs {
```

Comment:
this looking pretty good but is missing the media queries. you should be able to grab those from the baltcogo-reporter css

Reply:
yup I forgot about those. Will add them now

Reply:
added